applications/nrf5340_audio/unicast_server/main.c

```diff
@@ -482,10 +482,10 @@ void streamctrl_send(void const *const data, size_t size, uint8_t num_ch)
 int main(void)
 {
 	int ret;
+	static struct bt_data ext_adv_buf[CONFIG_EXT_ADV_BUF_MAX];
```

Comment:
Same here.

Reply:
Created another PR for that:
https://github.com/nrfconnect/sdk-nrf/pull/14217